themegrill/colormag
# Fixed primary menu hover color issue

What is done in this PR?

- Apply primary color to primary menu item hover color on selecting primary color.

How to test?
Go to customize>global>primary colors and select desired primary color.
The primary menu item hover color should change.
